pkgs/misc/toggl/default.nix

```diff
@@ -0,0 +1,27 @@
+{ qtbase, poco, fetchFromGitHub, stdenv, makeQtWrapper, perl, readline,
+  qtx11extras, qtwebkit, qmakeHook ,uncrustify, libXScrnSaver, scrnsaverproto }:
+stdenv.mkDerivation {
+  name = "toggl";
+  buildInputs = [qtbase poco perl readline qtwebkit qtx11extras uncrustify libXScrnSaver scrnsaverproto];
+  nativeBuiltInputs = [makeQtWrapper];
+  src = fetchFromGitHub {
+    repo = "toggldesktop";
+    owner = "toggl";
+    sha256="0x9iivb6ixvb4rc5h22pyqvgh3jy7114pijp87n8yyw209ahz690";
+    rev="v7.4.45";
+  };
+  enableParallelBuilding=true;
+  installFlags=["INSTALL_ROOT=$(out)"];
+  qmakeFlags=["PREFIX=/"];
+  prePatch = ''
+    sed -i 's@/usr/local/bin/@@' ./third_party/Xcode-formatter/CodeFormatter/scripts/formatAllSources.sh
+  '';
+  preBuild = ''
+    command -v perl
```

Comment:
it's probably not, I had an issue when build mechanism wanted perl  but couldn't find that, so I added this line to make sure it's there. I'll def remove that before merge, after this is "buildable".